### DISCUSSION TITLE
Override 'Resume Your Session' Modal trigger

### DISCUSSION THREAD

**Discussion by JorisOrangeStudio** [statamic/cms#8776] [Q&A] [answered]
Hello,
I'm curious how the 'Resume your Session' modal with password field is triggered. Does anyone one where I can find it? Since I want to redirect that modal to the login page instead of showing the modal (because we are using custom login controller). I already found the LoginModal.vue, but what is the best way of overriding this behaviour inside an addon?

**Comment by jasonvarga** [comment, marked as answer]
The Vue component is in `partials/session-expiry.blade.php`.

You can prevent it for now by adding an empty `resources/views/vendor/statamic/partials/session-expiry.blade.php` file.

A better long term solution would be a PR to include a setting to disable the feature.
